data/remnant/remnant ships.txt

```diff
@@ -1035,6 +1035,60 @@ ship "Puffin"
 
 
 
+ship "Smew"
+	sprite "ship/smew"
+	thumbnail "thumbnail/smew"
+	attributes
+		category "Fighter"
+		licenses
+			Remnant
+		"cost" 1641000
+		"shields" 1200
+		"hull" 2800
+		"cargo space" 90
+		"fuel capacity" 100
+		"ramscoop" 0.8
+		"required crew" 1
+		"bunks" 9
+		"mass" 60
+		"drag" 1.96
+		"heat dissipation" .65
+		"outfit space" 92
+		"weapon capacity" 10
+		"engine capacity" 42
```

Comment:
```suggestion
		"engine capacity" 46
```

This allows for a bit more flexibility in engine outfitting; specifically it allows an Anvil + Crucible Thruster combination. As this was originally a lifeboat, getting *away* from stuff takes precedence over manoeuvrability, so the Anvil + Crucible is the best possible engine configuration it could have.